mod/05_standard/set/self/account_links.rb

```diff
@@ -5,8 +5,9 @@
     #ENGLISH
     links = []
     if Auth.signed_in?
-      links << link_to_page( Auth.current.name, nil, :id=>'my-card-link' )
+      links << build_link( Auth.current.cardname, nil, :id=>'my-card-link' )
```

Comment:
Understood.  I was suggesting:
````
def build_link href, options
````
...but that if you had link text, you stick it in the options hash like:
````
build_link http://wagn.org, :text=>'the Wagn site'
````
Actually, I was playing with names other than build_link (link_to?), but you get the point.